app/client/keybase/slip_test.go

```diff
@@ -0,0 +1,308 @@
+		args        args
+		wantPrivHex string
+		wantPubHex  string
+
+		wantErr bool
+	}{
+		{
+			name: "Key(m) – master key",
+			args: args{
+				path: "m",
+				seed: seed,
+			},
+			wantPrivHex: "2b4be7f19ee27bbf30c667b642d5f4aa69fd169872f8fc3059c08ebae2eb19e7",
+			wantPubHex:  "00a4b2856bfec510abab89753fac1ac0e1112364e7d250545963f135f2a33188ed",
+
+			wantErr: false,
+		},
+		{
+			name: "Key(m/0')",
+			args: args{
+				path: "m/0'",
+				seed: seed,
+			},
+			wantPrivHex: "68e0fe46dfb67e368c75379acec591dad19df3cde26e63b93a8e704f1dade7a3",
+			wantPubHex:  "008c8a13df77a28f3445213a0f432fde644acaa215fc72dcdf300d5efaa85d350c",
+
+			wantErr: false,
+		},
+		{
+			name: "Key(m/0'/1')",
+			args: args{
+				path: "m/0'/1'",
+				seed: seed,
+			},
+			wantPrivHex: "b1d0bad404bf35da785a64ca1ac54b2617211d2777696fbffaf208f746ae84f2",
+			wantPubHex:  "001932a5270f335bed617d5b935c80aedb1a35bd9fc1e31acafd5372c30f5c1187",
+
+			wantErr: false,
+		},
+		{
+			name: "Key(m/0'/1'/2')",
+			args: args{
+				path: "m/0'/1'/2'",
+				seed: seed,
+			},
+			wantPrivHex: "92a5b23c0b8a99e37d07df3fb9966917f5d06e02ddbd909c7e184371463e9fc9",
+			wantPubHex:  "00ae98736566d30ed0e9d2f4486a64bc95740d89c7db33f52121f8ea8f76ff0fc1",
+
+			wantErr: false,
+		},
+		{
+			name: "Key(m/0'/1'/2'/2')",
+			args: args{
+				path: "m/0'/1'/2'/2'",
+				seed: seed,
+			},
+			wantPrivHex: "30d1dc7e5fc04c31219ab25a27ae00b50f6fd66622f6e9c913253d6511d1e662",
+			wantPubHex:  "008abae2d66361c879b900d204ad2cc4984fa2aa344dd7ddc46007329ac76c429c",
+
+			wantErr: false,
+		},
+		{
+			name: "Key(m/0'/1'/2'/2'/1000000000')",
+			args: args{
+				path: "m/0'/1'/2'/2'/1000000000'",
+				seed: seed,
+			},
+			wantPrivHex: "8f94d394a8e8fd6b1bc2f3f49f5c47e385281d5c17e65324b0f62483e37e8793",
+			wantPubHex:  "003c24da049451555d51a7014a37337aa4e12d41e485abccfa46b47dfb2af54b7a",
+
+			wantErr: false,
+		},
+		{
+			name: "Key(invalid)",
+			args: args{
+				path: "m/0",
+				seed: seed,
+			},
+			wantPrivHex: "",
+			wantErr:     true,
+		},
+	}
+	for _, tv := range tests {
+		t.Run(tv.name, func(t *testing.T) {
+			childKey, err := crypto.DeriveChild(tv.args.path, tv.args.seed)
+			if (err != nil) != tv.wantErr {
+				t.Errorf("DeriveChild() error = %v, wantErr %v", err, tv.wantErr)
+				return
+			}
+
+			if err != nil {
+				return
+			}
+
+			// Slip-0010 private keys in test vector are only the seed of the full private key
+			// This is equivalent to the SecretKey of the HMAC key used to generate the ed25519 key
+			privSeed, err := childKey.GetSeed("")
+			require.NoError(t, err)
+			privHex := hex.EncodeToString(privSeed)
+			require.Equal(t, privHex, tv.wantPrivHex)
+
+			// Slip-0010 keys are prefixed with "00" in the test vectors
+			pubHex := childKey.GetPublicKey().String()
+			require.Equal(t, "00"+pubHex, tv.wantPubHex)
+		})
+	}
+}
+
+// https://github.com/satoshilabs/slips/blob/master/slip-0010.md#test-vector-2-for-ed25519
+func TestSlip_DeriveChild_Vector2(t *testing.T) {
```

Comment:
You can just consolidate these two tests into one and change the params.  That's what the structs & forloop are setup for.